torchvision/models/quantization/mobilenetv3.py

```diff
@@ -79,11 +81,12 @@ def _load_weights(
     model: QuantizableMobileNetV3,
     model_url: Optional[str],
     progress: bool,
+    strict: bool
```

Comment:
nit: default to True, which is desired in most of cases.

Reply:
Indeed we always pass strict=True. This here is out of necessity. See my earlier comment at https://github.com/pytorch/vision/pull/4487#discussion_r717030133

Reply:
I was referring to setting a default value here like `strict: bool=True` so that you don't need to explicitly set strict=True in line 119.
BTW, for readability, I always use name argument if the value is primitive.

Reply:
Oh I gotcha, sounds good. I'll make the change.